src/functions-and-arrays.js

```diff
@@ -78,7 +185,15 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(wordsCount, anyWord) {
+  let counter = 0
+  for (let i = 0; i < wordsCount.length; i++) {
+    if (wordsCount[i] === anyWord) {
+      counter++
+    }
+  }
```

Comment:
Para esta clase de iteraciones simples, te recomiendo que te sirvas o bien de un bucle "for of·" o del método "forEach":
`for(word of wordsCount) {...}`
`wordsCount.forEach(word => ...)`
Pronto dejaréis de usar los bucles tradicionales (o al menos no los usaréis tan a menudo) y conviene acostumbrarse a estos métodos lo antes posible.